app/_src/policies/meshpassthrough.md

```diff
@@ -0,0 +1,155 @@
+---
+title: MeshPassthrough
+---
+
+{% warning %}
+This policy uses new policy matching algorithm.
+{% endwarning %}
+
+This policy enables {{site.mesh_product_name}} to configure external traffic for specific sidecar.
+When using this policy, the [passthrough mode](/docs/{{ page.version }}/networking/non-mesh-traffic/#outgoing) flag is ignored.
+
+## TargetRef support matrix
+
+{% tabs targetRef useUrlFragment=false %}
+{% tab targetRef Sidecar %}
+| `targetRef`           | Allowed kinds         |
+| --------------------- | --------------------- |
+| `targetRef.kind`      | `Mesh`, `MeshSubset`  |
+{% endtab %}
+{% endtabs %}
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+{% warning %}
+This policy doesn't work with sidecars not using [transparent-proxy](/docs/{{ page.version }}/networking/transparent-proxying/#what-is-transparent-proxying).
+{% endwarning %}
+
+The following describes the default configuration settings of the `MeshPassthrough` policy:
+
+- **`enabled`**: (Optional) If true, allows traffic to all external destinations.
+- **`appendMatch`**: List of destinations that sidecar should be able to reach. When `enabled` is `true` this list is not used. It works only when `enabled` is `false`.
+  - **`type`**: Defines what type of destination should be allowed. Either `Domain`, `IP` or `CIDR`
+  - **`value`**: Destination address based on the defined `type`.
+  - **`port`**: Port at which exterbal service is available. When not defined it caches all traffic to the address.
+  - **`protocol`**: Defines protocol of the external service.
+    - **`tcp`**: **Can't be used when `type` is `Domain` (at TCP level we are not able to disinguish domain, in this case it is going to hijack whole traffic on this port)**.
+    - **`tls`**: Should be used when TLS traffic is originated by the client application.
+    - **`http`**
+    - **`http2`**
+    - **`grpc`**
+  
+### Wildcard domains
+
+A `MeshPassthrough` policy allows you to create a match for a wildcard domain. You can match a subdomain and allow this traffic to go outside of the cluster.
+
+{% warning %}
+Currently, partial matches are not possible. For example, the domain `*ww.example.com` will be rejected.
```

Comment:
I changed it to `*w` because it's 2 characters long and it's harder to mix up with just `www`. (`*ww` vs `www`)

```suggestion
Currently, support for partial subdomain matching is not implemented. For example, a match for `*w.example.com` will be rejected.
```